core/src/main/resources/hudson/model/View/newJob.jelly

```diff
@@ -65,7 +65,7 @@ THE SOFTWARE.
             </j:if>
 
             <div class="footer">
-              <div class="btn-decorator">
+              <div class="yui-button">
```

Comment:
@recena the button is in a `footer` class and hence it doesn't get the YUI style, like the buttons from the other views i.e. the job config step. Here the buttons are part of the table and in the `additem.less` there is reference to the YUI styles.